### DISCUSSION TITLE
The storage optimization attempt failed. Storage optimization may be temporarily disrupted if manual event deletion is requested.

### DISCUSSION THREAD

**Discussion by al-navidh** [datalust/seq-tickets#2272] [Help]
I am on SEQ version 2024.3.12250 running on a Windows server 2019 VM and I get the attached error. How would go about fixing this error?
![Screenshot 2024-09-11 084019](https://github.com/user-attachments/assets/8237481c-23fd-4392-98aa-7985f73a62ac)

**Comment by nblumhardt** [comment]
Hi @al-navidh; the diagnostic report (Settings > Diagnostics > Download a full diagnostic report) will contain more information about the issue. If you need help interpreting it, please post the related exception message here, or if you have a supported Seq subscription, feel free to shoot this through to `support@datalust.co` for a closer look. Hope this helps!
